src/components/common/PageLayout/index.tsx

```diff
@@ -29,14 +31,14 @@ const PageLayout = ({ children }: { children: ReactElement }): ReactElement => {
       </header>
 
       {/* Desktop sidebar */}
-      <aside className={css.sidebar}>{sidebar}</aside>
+      {showSidebar && <aside className={css.sidebar}>{sidebar}</aside>}
 
       {/* Mobile sidebar */}
       <Drawer variant="temporary" anchor="left" open={isMobileDrawerOpen} onClose={onMenuToggle}>
         {sidebar}
       </Drawer>
 
-      <div className={css.main}>
+      <div className={`${css.main} ${!showSidebar ? css.mainNoSidebar : ''}`}>
```

Comment:
Please use `classnames`.
E.g. `className={classnames(css.main, {[css.mainNoSidebar]: !showSidebar })}`

Reply:
:+1: I didn't see it was there, btw there's a classnames alternative with a smaller bundle footprint, we might want to switch to that: https://www.npmjs.com/package/clsx